types/cordova-plugin-inappbrowser/cordova-plugin-inappbrowser-tests.ts

```diff
@@ -5,7 +5,15 @@
 // is similar to native window.open signature, so the compiler can's
 // select proper overload, but we cast result to InAppBrowser manually.
 var iab = <InAppBrowser>window.open('google.com', '_self');
-iab.addEventListener('loadstart', (ev: InAppBrowserEvent) => { console.log('Start opening ' + ev.url); });
+iab.addEventListener('loadstart', (ev) => { console.log('loadstart' + ev.url); });
```

Comment:
Is there a reason the type annotation was removed from the callback param on every test? Does that have an impact? If so, does this mean this change may not work with existing code? If not, would be good to leave in some places for extra coverage.